src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -129,4 +129,62 @@ tnsr::ii<DataType, SpatialDim, Frame> extrinsic_curvature(
 template <size_t SpatialDim, typename Frame, typename DataType>
 tnsr::ijj<DataType, SpatialDim, Frame> deriv_spatial_metric(
     const tnsr::iaa<DataType, SpatialDim, Frame>& phi) noexcept;
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes spatial derivatives of lapse (N) from the generalized
+ *        harmonic variables and spacetime unit normal 1-form.
+ *
+ * \details If the generalized harmonic conjugate momentum and spatial
```

Comment:
Same comments about as for the time deriv one below :)

Reply:
Just added KerrSchild test.